rqt_gui_cpp/CMakeLists.txt

```diff
@@ -1,20 +1,37 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(rqt_gui_cpp)
-# Load catkin and all dependencies required for this package
-find_package(catkin REQUIRED COMPONENTS qt_gui qt_gui_cpp roscpp nodelet)
-catkin_package(
-	INCLUDE_DIRS include
-	LIBRARIES ${PROJECT_NAME}
-	CATKIN_DEPENDS qt_gui qt_gui_cpp roscpp nodelet
-)
 
-include_directories(include ${catkin_INCLUDE_DIRS})
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# Qt5 boilerplate options from http://doc.qt.io/qt-5/cmake-manual.html
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_AUTOMOC ON)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
```

Comment:
Please use `add_compile_options` instead. I don't think it is supported on all platforms?

Reply:
This is from https://github.com/ros-visualization/qt_gui_core/blob/def6b8d4e28d42523bcd872729d3d2720dc821fb/qt_gui_cpp/src/qt_gui_cpp/CMakeLists.txt#L5

@brawner Did you put this here and do we need to change it to use `add_compile_options`?

Reply:
Can't blame me for this one